lib/bolt/cli.rb

```diff
@@ -326,6 +326,16 @@ def read_arg_file(file)
       raise Bolt::CLIError, "Error attempting to read #{file}: #{err}"
     end
 
+    # This regex should be in one place so we don't have to keep track of it
+    def validate_task_name(name)
```

Comment:
This doesn't seem like what Alex had in mind for regexp validation, but I wanted to use what I know to figure out where to validate first, and keep the regexp in one place.